oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +343,38 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+        String[] statSplit = stat.split(" ");
```

Comment:
For splitting on whitespace, use `Parseutil.whitespace.split()` which precompiles this.  However, I believe some filenames can have spaces in them, so this won't work as easily as you've implemented. You need to find the open and close parens and read the values between them. `String.indexOf()` for `(` and `)` should give you enough info to grab all the appropriate substrings without using split.